ui/vty-app/Glue.hs

```diff
@@ -253,7 +254,8 @@ knitCommandResultToUI commandId = Just . UiCommandEvent commandId . \case
   KnitCommandProcError e ->
     UiCommandFailure $ Knit.ppResolveErrors e
   KnitCommandException e ->
-    UiCommandFailure $ PP.text (displayException e)
+    UiCommandFailure $ PP.string $
+      toString $ wrapText defaultWrapSettings 80 (fromString $ displayException e)
```

Comment:
doesn't look really good, will be glad if you can suggest better solution for auto line breaking repl messages

Reply:
I'd ask @goodsoft, I don't know much about TUI drawing.

Reply:
Why is this needed at all? When `doc` is rendered to the screen `pprDoc` should automatically wrap it to the screen width. Why do you want to wrap it to 80 symbols?